### PR TITLE
Rename asset extrinsic

### PR DESCRIPTION
**Pull request by vovac12**

### CODE REVIEW

File: pallets/assets/src/tests.rs [hunk 1]
```diff
@@ -795,4 +795,106 @@ mod tests {
...
+                (
+                    AssetSymbol(b"PSWAP".to_vec()),
+                    AssetName(b"Polkaswap".to_vec()),
+                    DEFAULT_BALANCE_PRECISION,
+                    true,
+                    None,
+                    None
+                )
+            );
+        });
```
Comment (wer1st) [inline pallets/assets/src/tests.rs:898]: Add tests with empty name & symbol please

Comment (wer1st) [reply pallets/assets/src/tests.rs:898]: And test for non-existing asset

File: pallets/assets/src/lib.rs [hunk 2]
```diff
@@ -460,6 +461,36 @@ pub mod pallet {
...
+                    ensure!(new_name.is_valid(), Error::<T>::InvalidAssetName);
+                    *name = new_name;
+                }
+                if let Some(new_symbol) = new_symbol {
+                    ensure!(new_symbol.is_valid(), Error::<T>::InvalidAssetSymbol);
+                    *symbol = new_symbol;
+                }
+                DispatchResult::Ok(())
+            })?;
+            Ok(().into())
```
Comment (bragov4ik) [inline pallets/assets/src/lib.rs:492]: Just wondering if it makes sense to emit event here, like on asset registration

**Review by wer1st — APPROVED** [review]

**Review by bragov4ik — APPROVED** [review]
